template/backend/dapp/assets/{{name}}/contracts/{{name}}.hbs.sol

```diff
@@ -1,4 +1,3 @@
-pragma solidvm 3.4;
 
```

Comment:
small thing - but do you mind removing the empty lines at the top too? so the import is the top line (same for the other files)